microsoft-edge/webview2/samples/webview2apissample.md

```diff
@@ -1,133 +1,142 @@
 ---
-title: Win32 sample app
+title: Win32 sample app (WebView2APISample)
```

Comment:
Let's keep the title to just "Win32 sample app" or else it's too long and takes up a lot of space in the TOC

Reply:
changed globally